src/nunjucks/moduk/components/error-summary/__examples__/with-link-error-to-input.njk

```diff
@@ -0,0 +1,45 @@
+{% from "moduk/components/error-summary/macro.njk" import modukErrorSummary %}
+{% from "moduk/components/date-input/macro.njk" import modukDateInput %}
+
+<div>
+  {{ modukErrorSummary({
+    titleText: "There is a problem",
+    errorList: [
+      {
+        text: "The start date of the contract must include a year",
+        href: "#incorect-date-format-year"
+      }
+    ]
+  }) }}
+
+  {{ modukDateInput({
+    id: "incorect-date-format",
+    namePrefix: "incorect-date-format",
+    fieldset: {
+      legend: {
+         isPageHeading: true,
+        text: "When will the contract start?",
+        classes: 'govuk-fieldset__legend--l'
+      }
+    },
+    errorMessage: {
+      text: "The start date of the contract must include a year"
+    },
+    items: [
+      {
+          classes: "govuk-input--width-2",
+          name: "day",
+          value: "6"
+      },
+      {
+          classes: "govuk-input--width-2",
+          name: "month",
+          value: "3"
+      },
+      {
+          classes: "govuk-input--width-4 govuk-input--error",
+          name: "year"
+      }
+    ]
+  }) }}
+</div>
```

Comment:
Any idea for a better filename?

Reply:
Hmm `with-date-part-error.njk` or `with-error-linked-to-date-part.njk` maybe